var/spack/repos/builtin/packages/datatransferkit/package.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Datatransferkit(CMakePackage):
+    """DataTransferKit is an open-source software library of
+    parallel solution transfer services for multiphysics simulations"""
+
+    homepage = "https://datatransferkit.readthedoc.io"
+    url      = "https://github.com/ORNL-CEES/DataTransferKit/archive/3.1-rc1.tar.gz"
+    git      = "https://github.com/ORNL-CEES/DataTransferKit.git"
+
```

Comment:
@Rombur would you like to be added as a maintainer?
```python
maintainers = ['Rombur']
```
This gives us someone to ping when anyone reports build issues with this package or submits a PR to modify the build recipe.

Reply:
Sure